examples/js/hallo.mjs

```diff
@@ -0,0 +1,163 @@
+/* Generated by the Nim Compiler v1.0.6 */
```

Comment:
generated files should not be checked in

Reply:
Sure, it's a WIP. Should be squashed eventually. I'll remove it now.  Just raised an issue here.
Would be nice with better (or explicit?) support for ES modules.

https://github.com/nim-lang/Nim/issues/13297

Main issue is that any string proc argument is escaped in the output as a string it seems.

Reply:
alternatively you can  `git rebase -i HEAD~3` (for example) to cleanup your PR commit history and git push --force